recipes/cctk/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
   sha256: {{ hash }}
 
 build:
-  number: 0
+  number: 1
```

Comment:
But why are you bumping the build number if there is no new release?

Reply:
The build failed the Lint step at [ab2f11c](https://github.com/bioconda/bioconda-recipes/pull/52886/commits/ab2f11cbf137b29a60dc8f4c649be45aa8d4ef10) with an error saying the build number needed to be bumped.

Reply:
https://anaconda.org/bioconda/cctk/files version 1.0.3 already exists. What do you want to do with this PR :)

Reply:
Wow. I should have checked. The bot didn't start the autobump when I made the release in December (presumably because of the namespace issue?) so I had assumed I would need to finish this PR.

We can close this. Thank you for the help!